core/embed/models/D002/vendorheader/vendor_dev_DO_NOT_SIGN.json

```diff
@@ -2,6 +2,7 @@
     "header_len": 4608,
     "text": "DEV ONLY, DO NOT USE!",
     "hw_model": "D002",
+    "fw_type": 2,
```

Comment:
I am not clear on what's the difference between UNSAFE and dev_DO_NOT_SIGN. Is UNSAFE what we normally refer to as custom firmware on normal end-user devices? What's the latter then? Is it firmware that can be run only on developer devices that have a special boardloader/bootloader?

The reason why I ask is that it seems to me that `fw_type` should be associated with the pubkeys, in which case these `fw_type`s for dev_DO_NOT_SIGN would be 1. However if these vendor headers are usable only on special developer devices, then the current numbers make sense.